pulsar-client-cpp/include/pulsar/c/client.h

```diff
@@ -90,11 +90,22 @@ PULSAR_PUBLIC void pulsar_client_subscribe_async(pulsar_client_t *client, const
                                                  const pulsar_consumer_configuration_t *conf,
                                                  pulsar_subscribe_callback callback, void *ctx);
 
+PULSAR_PUBLIC pulsar_result pulsar_client_subscribe_multi_topics(pulsar_client_t *client, const char **topics,
```

Comment:
Do not forget to add descriptions (annotation) so that users know how to use them.